src/utils.ts

```diff
@@ -53,7 +49,9 @@ export function parseNgTemplateBindings(input: string) {
   const { templateBindings: ast, errors } = ngParser.parseTemplateBindings(
     NG_PARSE_TEMPLATE_BINDINGS_FAKE_PREFIX,
     input,
-    ...NG_PARSE_SHARED_PARAMS,
+    NG_PARSE_FAKE_LOCATION,
+    NG_PARSE_FAKE_ABSOLUTE_OFFSET,
+    NG_PARSE_FAKE_ABSOLUTE_OFFSET,
```

Comment:
Though we alway pass `0`, but better define `NG_PARSE_FAKE_ABSOLUTE_KEY_OFFSET` and `NG_PARSE_FAKE_ABSOLUTE_VALUE_OFFSET`,or rename that constant.

Reply:
PRs welcome :)